lib/menu.rb

```diff
@@ -0,0 +1,58 @@
+class Menu
+  attr_reader :order, :total, :sum
+
+  def initialize
+    @order = []
+    @sum = 0
+    @total = 0
+  end
+
+  def list_food
+    @menu = {
+      "pizza" => 10,
+      "lasagna" => 8,
+      "rigatoni" => 15,
+      "spaghetti" => 7,
+      "salad" => 5,
+      "burrata" => 6
+    }
+  end
+
+  def correct?
+    @total == @sum
+  end
+
+  def make_order
```

Comment:
Do you think it is the responsibility of the menu class to place the order, do the basket and sum of it ?
Maybe an other class to make your order like Order or Basket class would have been a way to approach the challenge.